stringtable/genstringtable.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
Commenting on the name of the file:

We have an `internal/cli` directory where this might make a lot of sense.